src/pombot/commands/help.js

```diff
@@ -0,0 +1,11 @@
+/*
+ * A command to show help information.
+ */
+import {createCommand} from 'chatter';
+
+export default createCommand({
```

Comment:
And `name` is optional for the top-level command. You only want to give the top-level command a name if you don't want `help` potentially displaying for every message. Like when the command is in a public channel.